apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -78,11 +78,9 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
         // Must be <= supportRequiredPct to avoid votes being impossible to pass
         uint64 minAcceptQuorumPct;
 
-        // Note: we could re-work these concepts to be against the start of the vote rather than the end
```

Comment:
In terms of this, should we also "flip" the current concept we have of the quietEndingPeriod to be "since the start of a vote"?

Reply:
We discussed offline; actually this the quietEndingPeriod makes more sense being calculated from the original end date.

Reply:
As discussed offline, since the logic won't be simplified, we agreed on keeping this setting tied to the vote end date. Additionally, it feels clearer from a user perspective